app/src/main/java/com/example/puzzlepic/ui/puzzle/PuzzleFragment.kt

```diff
@@ -27,12 +27,12 @@ import java.util.*
 
 class PuzzleFragment : Fragment() {
 
+    private var _binding: PuzzleFragmentBinding? = null
+    private val binding get() = _binding!!
+    private lateinit var currentPhotoPath: String
```

Comment:
Reordering the modifiers and putting them in the correct order.
Source:https://kotlinlang.org/docs/coding-conventions.html#modifiers-order